src/main/java/com/thealgorithms/dynamicprogramming/TravellingSalesMan.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Arrays;
+
+public class TravellingSalesMan {
```

Comment:
From my understand this class could be a _proper utility_,

```suggestion
public final class TravellingSalesMan {
    private TravellingSalesMan() {
    }
```